### PR TITLE
RD-4279: Added 'Show first user journey buttons' option to the documentation

### PR DESCRIPTION
**Pull request by NorbertAmartus**
Documentation is related to the changes introduced by [this PR](https://github.com/cloudify-cosmo/cloudify-stage/pull/1883)
Synchronization should be done as a part of the linked [PR](https://github.com/cloudify-cosmo/cloudify-stage/pull/1883)

### CODE REVIEW

File: content/working_with/console/widgets/deployments.md [hunk 1]
```diff
@@ -161,5 +161,11 @@ By default, that page displays the following:
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
 * `Enable click to drill down` - This option enables redirecting to the deployment’s drill down page upon clicking on a specific deployments. Default: On
 * `Show execution status label` - Allows showing last execution workflow ID and status near last execution status icon. Default: Off
+* `Show first user journey buttons` - When there are no installed deployments, it allows showing simplified view,  as shown below.
```
Comment (kubama) [inline content/working_with/console/widgets/deployments.md:164]:
```suggestion
* `Show first user journey buttons` - When there are no installed deployments, it presents a simplified view, as shown below.
```

**Review by kubama — APPROVED** [review]

File: content/working_with/console/widgets/deployments.md [hunk 2]
```diff
@@ -161,5 +161,11 @@ By default, that page displays the following:
 * `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
 * `Enable click to drill down` - This option enables redirecting to the deployment’s drill down page upon clicking on a specific deployments. Default: On
 * `Show execution status label` - Allows showing last execution workflow ID and status near last execution status icon. Default: Off
+* `Show first user journey buttons` - When there are no installed deployments, it presents a simplified view, as shown below.
+![First user journey buttons]( /images/ui/widgets/blueprint-deployments_first_user_journey.png)
+There are 2 buttons in the view :
```
Comment (qooban) [inline content/working_with/console/widgets/deployments.md:166]: There's small formatting issue :-) 

image

Comment (NorbertAmartus) [reply content/working_with/console/widgets/deployments.md]: Nice catch!
On the smaller screen sizes (like laptop) it seems to be displayed properly, that's why I didn't spoted it
I'll be aware of it in the future, to check docs with possibly the biggest screen available 😁 

Thanks for checking and pointing that out 🎉

Comment (NorbertAmartus) [reply content/working_with/console/widgets/deployments.md]: Necessary changes has been provided 🎸

**Review by qooban — APPROVED** [review]

**Pull request merged**